ext/openssl/openssl.c

```diff
@@ -5989,7 +5989,11 @@ PHP_FUNCTION(openssl_cms_verify)
 			}
 
 			if (p7bout) {
-				PEM_write_bio_CMS(p7bout, cms);
+				if (PEM_write_bio_CMS(p7bout, cms) == 0) {
+					php_error_docref(NULL, E_WARNING, "Failed to write CMS to BIO");
```

Comment:
Users might not be exactly aware what BIO is.

```suggestion
					php_error_docref(NULL, E_WARNING, "Failed to write CMS to file");
```